### PR TITLE
Pull request for documentation changes.

### PR DESCRIPTION
**Pull request by ronaldddilley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,15 +75,17 @@ See `land_surface_temperature.py --help` for command line details.
...
+The following input data are required to generate the Land Surface Temperature products:
+* Level 1 Product
+  - The thermal band from the level 1 product is also utilized.
 * Top of Atmosphere Reflectance (TOA)
   - TOA products can be generated using the [LEDAPS](https://github.com/USGS-EROS/espa-surface-reflectance) or [L8_SR](https://github.com/USGS-EROS/espa-surface-reflectance) software found in our [espa-surface-reflectance](https://github.com/USGS-EROS/espa-surface-reflectance) project.  Or through our ondemand processing system [ESPA](https://espa.cr.usgs.gov), be sure to select the ENVI output format.
 * Elevation
-  - Elevation data is expected to be in the same projection and image dimensions as the TOA products and in the ENVI file format.  The filename is also expected be `<sceneid>_dem.img`: where <b>sceneid</b> is the Landsat Scene ID of the data being processed.
+  - Elevation data is expected to be in the same projection, resolution, and image dimensions as the TOA products as well as in the ENVI file format.  The filename is also expected be `<sceneid>_dem.img`: where <b>sceneid</b> is the Landsat Scene ID of the data being processed.
```
Comment (gschmidt-usgs) [inline README.md]: Is it also required to be in the XML file?  Or is just the image required?

**Comment by gschmidt-usgs** [comment]
Peer review complete.

**Comment by raydittmeier** [comment]
I don't have any comments on the changed section, but the part right before it (on TOA Reflectance) probably could be reworded to flow a little better.  Maybe the "be sure" part could be a new sentence.  Maybe make "ondemand" "on-demand" or "on demand".  Maybe change "Or through our" to "TOA products can also be processed through our".

**Comment by ronaldddilley** [comment]
Updated both sections of the README file.

**Comment by raydittmeier** [comment]
The TOA Reflectance section changes look good, and I don't have any new comments on the rest.

**Pull request merged**